packages/code-du-travail-frontend/pages/_app.js

```diff
@@ -14,6 +14,8 @@ import { initializeSentry } from "../src/sentry";
 import { A11y } from "../src/a11y";
 
 import HeadBandAlert from "../src/common/HeadBandAlert";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import "katex/dist/katex.min.css";
```

Comment:
Est-ce que ce ne serait pas plus interessant d'import ces css là uniquement dans les outils (voir sur la denrière step uniquement ?)

Reply:
nope ca ne marche pas, l'import de css doit se faire depuis _app.js

Reply:
Lorsqu'on essaye d'importer des CSS de cette manière on ne peut le faire que dans _app.js :

> These styles (styles.css) will apply to all pages and components in your application. Due to the global nature of stylesheets, and to avoid conflicts, you may only import them inside pages/_app.js.

https://nextjs.org/docs/basic-features/built-in-css-support